.storybook/recipes/GlobalHeader/GlobalHeader.tsx

```diff
@@ -64,7 +64,27 @@ export const GlobalHeader = ({ className, ...other }: Props) => {
   });
   return (
     <Header behavior="sticky" className={componentClassName} {...other}>
-      <Logo className={styles['global-header__logo']} href="#" />
+      <Logo
+        ariaLabel="Link to Learning Platform Homepage"
```

Comment:
"Link to" seems redundant since the screen reader will announce that it's a link. Maybe just "Learning Platform Homepage"? I think I've heard from accessibility trainers before that it can just be the app name, so just "Learning Platform".